src/smt/smt_engine.cpp

```diff
@@ -2050,6 +2052,10 @@ void SmtEngine::setDefaults() {
     if( !options::miniscopeQuantFreeVar.wasSetByUser() ){
       options::miniscopeQuantFreeVar.set( false );
     }
+    if (!options::quantSplit.wasSetByUser())
+    {
+      options::quantSplit.set(false);
```

Comment:
did you evaluate the impact of removing this is default with CEGQI?

Reply:
This CEGQI is "sygus CEGQI". So if this ever kicked in before, we'd segfault :)